Automattic/wp-calypso
# Desktop App: Enable Gutenberg

### Changes proposed in this Pull Request

This PR enables the Gutenberg flags on the desktop environments, so the block editor becomes usable on the desktop app.

### Testing instructions

This needs to be tested on the desktop app, so make sure you have the dev environment [installed and running](https://github.com/Automattic/wp-desktop/blob/b729377828ec53f4469ed27a15c1fbc6e9b3a1cf/docs/install.md) (more info about how to set up the secrets in paaHJt-7y-p2).

* Apply D25762-code to your sandbox site.
* In the desktop repo, point the calypso submodule to this branch.
* Create a build: `make build`.
* Open the built app in the `release` folder.
* Switch to your sandbox site.
* Make sure that Gutenberg works like in the production environment:
  * Check opt-in and opt-out flows (making sure that the opt-out flow redirects to the classic editor which is opened inside the desktop app).
  * Try publishing a new post.
  * Test the Gutenframe integration (media modal, revision, move to trash).
  * Confirm that the back to the posts button redirects to the lists of posts page and it is opened inside the desktop app.
  * Try editing an existing post.
